package.json

```diff
@@ -49,17 +49,15 @@
     "@jest/types": "^29.0.0",
     "@octokit/tsconfig": "^1.0.1",
     "@octokit/webhooks-schemas": "6.10.0",
-    "@pika/pack": "^0.3.7",
-    "@pika/plugin-build-node": "^0.9.2",
-    "@pika/plugin-build-web": "^0.9.2",
-    "@pika/plugin-ts-standard-pkg": "^0.9.2",
```

Comment:
We need to update `.gitignore` as well.

Reply:
Done in 7d570d736f90104e09ed029ccb95a6d84de72766